components/automate-ui/src/app/pages/+compliance/+profile/+profile-overview/profile-overview.component.ts

```diff
@@ -262,11 +264,24 @@ export class ProfileOverviewComponent implements OnInit, OnDestroy {
     observableForkJoin(installProfiles).pipe(
       takeUntil(this.isDestroyed))
       .subscribe(
-        () => this.refreshProfiles(),
-        () => this.showDownloadError()
+      () => this.refreshProfiles(),
+      (error) => this.showError(error)
       );
   }
 
+  showError(error) {
+    if (error.status === HttpStatus.FORBIDDEN) {
+      this.authErrorVisible = true;
+    } else {
+      this.showDownloadError();
```

Comment:
I think it would be reasonable to inline this, here, too:
```suggestion
      this.downloadErrorVisible = true;
```
and remove `showDownloadError`. What do you think?